deploy/tests/_setup.ts

```diff
@@ -0,0 +1,51 @@
+import { isTenderlyFork } from '../../utils/Deploy';
+import axios from 'axios';
+import { config, network, tenderly } from 'hardhat';
+import { HttpNetworkUserConfig } from 'hardhat/types';
+
+interface EnvOptions {
+    TENDERLY_USERNAME: string;
+    TENDERLY_ACCESS_KEY: string;
+}
+
+const { TENDERLY_USERNAME, TENDERLY_ACCESS_KEY }: EnvOptions = process.env as any as EnvOptions;
+
+const TENDERLY_TEST_PROJECT_NAME = 'v3-temp-forks';
```

Comment:
shouldn't this be taken from the .env file?

Reply:
The rationale was to use a hard-coded "system project" that no one should configure, but I guess we can also move it to `.env` and specify it explicitly in workflow files

Reply:
Fixed